src/js/components/editor/Editor.vue

```diff
@@ -208,11 +114,8 @@ export default {
 
 <style lang="scss" scoped>
 
-$dkTan: #cd853f;
-$ltOrange: #f4a460;
-
 .text-pane {
-    overflow: hidden;
+    overflow: scroll;
```

Comment:
👍

Reply:
Yeah, that's gross. For the next round :)